lib/Textmaster/Model/ProjectInterface.php

```diff
@@ -270,6 +271,18 @@ public function createDocument();
     public function addDocuments(array $documents);
 
     /**
+     * Get all my authors who can do this project
+     *
+     * @param string|null $status Possible values: uncategorized, my_textmaster, blacklisted
+     *
+     * @return array
```

Comment:
I am wondering if this method should return array of `AuthorInterface` instead of just array of`author_id`.  
@cdaguerre wdyt?